web/app/tournament/tourny.py

```diff
@@ -85,3 +85,61 @@ def delete(tournament_id):
     # flash("Sorry, this tournament has already been submitted and can no \
     #       longer be edited.")
     return redirect(url_for('.index'))
+
+@tournament.route('/<int:tournament_id>/player/new/', methods=["GET", "POST"]) #  maybe a vanity url
```

Comment:
The 'GET' to this url returns the html form to create a new tournamentplayer.
Should I be thinking about render forms in a different way?